tasks/eth/018-granite-upgrade/.env

```diff
@@ -0,0 +1,9 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+OWNER_SAFE=0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+
+# Current nonce is 6 and 016 does not use this safe.
+SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=6
+# Current nonce is 7. However, 016 is sequenced to use the 7 nonce.
+SAFE_NONCE_0x847B5c174615B1B7fDF770882256e2D3E95b9D92=8
```

Comment:
Nit: the casing on the x in these addresses is inconsistent.
That appears [to be safe](https://github.com/base-org/contracts/blob/main/script/universal/NestedMultisigBuilder.sol#L63), but still would prefer we standardize.